File: pyproject2/moviedb.c
//mathew baird
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>
#include "scanner.h"
#include "support.h"

int
main(int argc, char **argv)
    {
    int countone,counttwo;
    if (argc != 3)
        {
        printf("need two arguments!\n");
        exit(1);
        }
    Movie **arrayone,**arraytwo,**finalarray;
    arrayone = readMovieRecordsList(argv[1],&countone);
    arraytwo = readMovieRecordsList(argv[2],&counttwo);
    finalarray = mergeAllArrays(arrayone,countone,arraytwo,counttwo);
    query(finalarray,countone+counttwo);
    return 0;
    }

File: pyproject2/moviedb.py
from scanner import *
from support import *
import sys


def main():
    if len(sys.argv) != 3:
        print("need two argvs")
        sys.exit(0)
    arrayone = readMovieRecordsList(sys.argv[1], countone)
    arraytwo = readMovieRecordsList(sys.argv[2], counttwo)
    finalarray = mergeAllArrays(arrayone, countone, arraytwo, counttwo)
    query(finalarray, countone + counttwo)

main()
